spec/integration/integration_spec.rb

```diff
@@ -901,5 +901,28 @@ class Tag
         end
       end
     end
+
+    describe 'overriden changes_method with non_database_fields' do
```

Comment:
The entire thing is to implement history tracking for embedded objects using nested attributes as the README suggests. Prior to these changes, that capability did not exist based on how it was described. The code simply didn't work because mongoid-history filtered out everything that wasn't a field.

If you take a look at the new implementation of tracked fields, I do believe that it is a large improvement over how it was implemented previously and gains you the ability to do non-database fields as a bonus.